protocol/0014-ORDT-order_types.md

```diff
@@ -370,8 +370,9 @@ In Spot market, for multiple iceberg orders submitted as a batch of orders with
   - Be triggered by a fall to `45`. (<a name="0014-ORDT-063" href="#0014-ORDT-063">0014-ORDT-063</a>)
   - Not be triggered by a fall to `46`. (<a name="0014-ORDT-064" href="#0014-ORDT-064">0014-ORDT-064</a>)
 
-- A stop order placed either prior to or during an auction will not execute during an auction, nor will it participate in the uncrossing. (<a name="0014-ORDT-065" href="#0014-ORDT-065">0014-ORDT-065</a>)
-- A stop order placed either prior to or during an auction, where the uncrossing price is within the triggering range, will immediately execute following uncrossing. (<a name="0014-ORDT-066" href="#0014-ORDT-066">0014-ORDT-066</a>)
+- A stop order placed either prior to or during a monitoring auction will not execute during an auction, nor will it participate in the uncrossing. (<a name="0014-ORDT-065" href="#0014-ORDT-065">0014-ORDT-065</a>)
+- A stop order placed either prior to or during a monitoring auction, where the uncrossing price is within the triggering range, will immediately execute following uncrossing. (<a name="0014-ORDT-066" href="#0014-ORDT-066">0014-ORDT-066</a>)
+- A stop order will be rejected if submitted while the market is in opening auction (<a name="0014-ORDT-132" href="#0014-ORDT-132">0014-ORDT-132</a>)
```

Comment:
there is an ac for this [0014-ORDT-133](https://github.com/vegaprotocol/specs/blob/palazzo/protocol/0014-ORDT-order_types.md#0014-ORDT-133)

Reply:
@Sohill-Patel - is this a direct duplicate or does 133 cover 132?